README.md

```diff
@@ -91,6 +91,39 @@ func main() {
 	errInterface := response[1].Interface()
 }
 ```
+Optionally, you can pass a custom name for your instrumented function:
+```
+	response := epsagon.GoWrapper(config, doTask, "<MyInstrumentedFuncName>")(5, "hello")
+```
```

Comment:
I think it could be nice to explain here the use case for specifying a function name